### PR TITLE
Family members

### PR DESCRIPTION
**Pull request by magas4891**
I added the ability to send an email with a link and connected resque to send SMS and email.

### CODE REVIEW

File: config/environments/development.rb [hunk 1]
```diff
@@ -73,5 +73,4 @@
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
   config.debug_exception_response_format = :default
-  config.action_mailer.default_url_options = { host: ENV.fetch('HOST'), port: 3000 }
```
Comment (drkmen) [inline config/environments/development.rb:76]: again, why did you remove it?

Comment (magas4891) [reply config/environments/development.rb:76]: it is on line 39

**Review by drkmen — APPROVED** [review]
